F1/F1/Race/Repository/CircuitDetails.swift

```diff
@@ -7,24 +7,23 @@
 
 import Foundation
 
-// MARK: - CircuitModel
-struct CircuitModel: Codable {
-    let mrData: CircuitData
+// MARK: - CircuitDetails
+struct CircuitDetails: Codable {
```

Comment:
make this singular please. Objects and models should always be singular. So CircuitDetail will do

Reply:
Genuinely thought about that and didn't know if "detail" made sense. Thanks for clearing it up